roaring/README.md

```diff
@@ -0,0 +1,30 @@
+# The Fuzzer
+
+For complete documentation on go-fuzz, please see: https://github.com/dvyukov/go-fuzz
+
+The fuzzer in relation to the roaring package checks the `Bitmap.UnmarshalBinary` function found in `roaring.go`. In order to use the fuzzer, you can follow these steps:
+
+`cd $GOPATH/src/github.com/pilosa/pilosa/roaring`
+
+`go-fuzz-build ./`
+
+You must now make the workdir/corpus directory. This is achieved by:
+
+`mkdir workdir/corpus`
+
+The fuzzer needs some input to start the fuzzing with. Copy some sample fragments into the workdir/corpus folder. Once you have copied your sample inputs, you are ready to run the fuzzer:
+
+`go-fuzz`
```

Comment:
can we add the `-bin`, `-workdir`, and `-func` flags here to be more explicit? The `func` and `workdir` flags are for supporting multiple fuzzing functions, which would each require different corpuses.